.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,21 +1,17 @@
 ---
-name: Feature request
-about: Suggest a new idea for Jamulus (not an improvement of an existing function)
+name: Feature
+about: Agreed specificaiton for a feature request or enhancement
 title: ''
-labels: 'feature request'
+labels: 'feature'
 assignees: ''
 ---
 
-<!-- Thanks for opening a new feature request! Please keep in mind that Jamulus achieves low latency audio by reducing complexity and following the "Keep it simple and stupid" approach. Adding new features may result in more processing and maintenance overhead, so please be sure that your feature can not be achieved by different approaches/software too. Make it clear, why here. -->
+<!-- Thanks for opening a new feature request! -->
 
-**Is your feature request related to a problem? Please describe.**
-<!--A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+**Has this feature been discussed and generally agreed?**
+<!-- Please ensure that your feature is a clearly specified piece of work that can be reasonably actioned by someone without too much help. If not, please post it in Discussions first so that it can become so. -->
 
 **Describe the solution you'd like**
-<! --A clear and concise description of what you want to happen. --> 
+<!-- A clear and concise description of what you want to happen. --> 
 
-**Describe alternatives you've considered**
```

Comment:
I wonder whether the alternatives part should be kept. When making technical decisions, it's useful to document why alternative solutions have not been implemented. Maybe it was intended to still have it but keep that part in the related discussion? Might be hard to "enforce" some guidance/structure there though.

Reply:
I agree. I'll add that back in (slightly re-worded to emphasise the hope that it's been discussed and alternatives thereby discounted).